projects/plugins/jetpack/sal/class.json-api-site-jetpack.php

```diff
@@ -696,4 +696,22 @@ public function get_wpcom_staging_blog_ids() {
 	public function get_wpcom_admin_interface() {
 		return null;
 	}
+
+	/**
+	 * Get Zendesk site meta. Not used in Jetpack.
+	 *
+	 * @return null
+	 */
+	public function get_zendesk_site_meta() {
+		return null;
```

Comment:
A nitpick, but in WP.com this always returns an array.

Reply:
🤷 I also note that the API endpoint using this was broken on self-hosted from April 2021 and October 2022, and no one noticed. Meaning probably no one actually uses it anyway.